spec/game_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'game'
+
+describe Game do
+  let(:game) { described_class.new }
+
+  describe '#initialize' do
+    it 'New games are initialized with a new frame' do
+      expect(game.frames).to be_an_instance_of(Array)
+    end
+  end
+
+  describe '#strike_bonus' do
+    it 'checks to see if last frame had a strike 1' do
+      game.roll(3)
+      game.roll(3)
+      game.roll(10)
+      game.roll(2)
+      expect(game.last_frame_strike?).to eq true
+    end
+    it 'checks to see if last frame had a strike 2' do
+      game.roll(3)
+      game.roll(3)
+      game.roll(10)
+      game.roll(3)
+      expect(game.last_frame_strike?).to eq true
+    end
+  end
+
+  describe '#score' do
+    it 'Takes into account strike bonus' do
+      game = Game.new
+      game.roll(10)
+      game.roll(3)
+      game.roll(4)
+      expect(game.frames[0].score).to eq 17
+      end
+    it 'Takes into account strike bonus' do
+      game = Game.new
+      game.roll(4)
+      game.roll(5)
+      game.roll(10)
+      game.roll(4)
+      game.roll(4)
+      expect(game.frames[1].score).to eq 18
+    end
+    it 'Takes into account strike bonus' do
+      game = Game.new
+      game.roll(10)
+      game.roll(5)
+      game.roll(1)
+      expect(game.frames[0].score).to eq 16
+    end
+    it 'Takes into account strike bonus' do
+      game = Game.new
+      game.roll(4)
+      game.roll(5)
+      game.roll(4)
+      game.roll(4)
+      game.roll(4)
+      game.roll(4)
+      game.roll(10)
+      game.roll(1)
+      game.roll(1)
+      expect(game.frames[3].score).to eq 12
+    end
```

Comment:
Are these tests necessary? 

It seems like they test the same scenario as the strike test [here](https://github.com/makersacademy/bowling-challenge-ruby/pull/19/files#diff-3f3416f9e912511dc1f61fc61d2832a11dae55401e24c04209cd84f7c40ad633R30-R36)